Jenkinsfile

```diff
@@ -248,32 +248,32 @@ pipeline {
         // - production deployment is only tested on Clear Linux
         //   and testing deployment only on Fedora
 
-        stage('testing 1.16') {
+        stage('testing 1.18') {
             options {
                 timeout(time: 180, unit: "MINUTES")
             }
             steps {
-                TestInVM("fedora", "", "1.16", "Top.Level..[[:alpha:]]*-production[[:space:]]")
+                TestInVM("fedora", "", "1.18", "Top.Level..[[:alpha:]]*-production[[:space:]]")
             }
         }
 
-        stage('testing 1.15') {
+        stage('testing 1.16') {
             when { not { changeRequest() } }
             options {
                 timeout(time: 180, unit: "MINUTES")
             }
             steps {
-                TestInVM("fedora", "", "1.15", "")
+                TestInVM("fedora", "", "1.16", "")
             }
         }
 
-        stage('testing 1.14') {
```

Comment:
We cannot just stop testing on 1.14 entirely without reporting that in the "Supported Kubernetes versions" section of the README.md. Let's do that in this PR, with this comment:
"Lacks support for ephemeral inline volumes. Not supported officially by upstream anymore."

Reply:
There are few other places 1.14 is mentioned in README, these need to be replaced with 1.15 I assume (and should verify these examples still work).
Also, once we touch "Supported Kubernetes Versions" table: this table currently goes up to 1.16, do we want to add 1.17 and 1.18 there, and do we need to add any "support status" about those versions?

Reply:
About "Supported table", we even have 1.13 mentioned there, should we remove?

Reply:
> do we want to add 1.17 and 1.18 there

Yes.

> do we need to add any "support status" about those versions

No, because there's nothing special to report.

> we even have 1.13 mentioned there, should we remove?

Let's keep it, to avoid questions along the lines of "do you support 1.13?" when we don't say anything about it.